src/wallet/wallet.cpp

```diff
@@ -1403,11 +1410,13 @@ bool CWallet::IsChange(const CTxOut& txout) const
     if (::IsMine(*this, txout.scriptPubKey))
     {
         CTxDestination address;
-        if (!ExtractDestination(txout.scriptPubKey, address))
+        const CScript& scriptPubKey = StripClaimScriptPrefix(txout.scriptPubKey);
```

Comment:
This seems wrong; we don't want to count our held claims as available change.

Reply:
https://github.com/lbryio/lbrycrd/blob/legacy_master/src/wallet/wallet.cpp#L1004